packages/calcite-components/src/components/input-date-picker/input-date-picker.e2e.ts

```diff
@@ -453,11 +454,13 @@ describe("calcite-input-date-picker", () => {
         await resetFocus(page);
         await startInput.click();
         await page.waitForChanges();
+        expect(openSpy).toHaveReceivedEventTimes(1);
```

Comment:
I should've spotted this before, but why are we asserting on the open/close events? Doesn't the `openClose` test helper cover this? Applies to `input-time-picker` as well.